pages/linux/nemo.md

```diff
@@ -1,16 +1,24 @@
 # nemo
 
-> File manager and graphical shell for Cinnamon.
-> More information: <https://github.com/linuxmint/nemo>.
+> Manages files and directories in Cinnamon desktop environment.
+> More information: <https://manned.org/nemo.1>.
 
-- Open a new window showing the user's home directory:
+- Open a current user home directory:
 
 `nemo`
 
-- Open a new window showing the current directory:
+- Open specific directories in separate windows:
```

Comment:
```suggestion
- Open one or more directories in separate windows:
```

Reply:
When we have to mention count of arguments (one or more in this case)?

Reply:
```suggestion
- Open directories in separate windows:
```
It's not a rule, just think it might read better. Perhaps this would be clearer since both specific, and one or more is obvious.